core/src/main/java/hudson/PluginManager.java

```diff
@@ -1521,11 +1521,11 @@ public HttpResponse doInstallPlugins(StaplerRequest req) throws IOException {
     }
 
     private void trackInitialPluginInstall(@Nonnull final List<Future<UpdateCenter.UpdateCenterJob>> installJobs) {
-        final Jenkins jenkins = Jenkins.getInstance();
+        final Jenkins jenkins = Jenkins.get();
         final UpdateCenter updateCenter = jenkins.getUpdateCenter();
         final Authentication currentAuth = Jenkins.getAuthentication();
 
-        if (!Jenkins.getInstance().getInstallState().isSetupComplete()) {
+        if (!Jenkins.getInstanceOrNull().getInstallState().isSetupComplete()) {
```

Comment:
Why? Would make the potential NPE explicit if it wouldn't already throw 4 lines up…?

Reply:
My suggestion was to keep using `jenkins` here. https://github.com/jenkinsci/jenkins/pull/4042#discussion_r287554871

Reply:
Sorry guys. I changed for use ```jenkins```instead